plugins/customrpc/components/RPCSettings.tsx

```diff
@@ -12,8 +12,17 @@ const {
   Category,
   ErrorBoundary,
   SwitchItem,
+  RadioItem
 } = components;
 
+const ActivityTypes = {
```

Comment:
would be better as enum ig

Reply:
Enums make it fucky-wucky as the professionals would say due to it being a two way mapped object. Current method works and doesnt result in.. this.
![image](https://github.com/E-boi/replugged-plugins/assets/46033729/c4d5318b-34c8-4c71-b688-abe377476ad5)